src/components/AboutUs/index.jsx

```diff
@@ -1,11 +1,17 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
 import './style.css';
+import React from 'react';
+import aboutUsData from '../../data/aboutUsData.json';
 
 function AboutUs() {
   return (
-    <div className="about-us-container">
-      <h1>About Us</h1>
+    <div className="jumbotron jumbotron-fluid">
+      <div className="container">
+        <h2 className="display-4">{aboutUsData.title}</h2>
+        <p className="lead">{aboutUsData.content}</p>
+      </div>
     </div>
```

Comment:
could you use react-bootstrap jumbotron componen, please? https://react-bootstrap-v3.netlify.app/components/jumbotron/

Reply:
you can also check container component if it fits

Reply:
done @ewelinasobora

Reply:
looks good